.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# These owners will be the default owners for everything in the repo.
+* @GlobeHoppin/admin
```

Comment:
This is not related to your changes right ?

Reply:
no i didn't changed this thing

Reply:
Yeah this is what happened with your command to squash.
You had 3 commits but you went 5 commits behind.

Follow my steps.

Reply:
what to do now chatgpt said this only

Reply:
can you once again confirm

Reply:
No, squashing does not revert the commits or move you back to a previous state. Instead, squashing combines multiple commits into a single commit without losing the changes those commits made.